assets/jsons/generosity_challenge/chat_scripts/chat_script_day_0.json

```diff
@@ -47,51 +47,57 @@
                   "next": {
                     "type": "textMessage",
                     "side": "interlocutor",
-                    "text": "Thanks! I've made a note of that and it's all stored safely",
+                    "text": "Thanks! I'll make a note of that and store it safely",
                     "next": {
                       "type": "textMessage",
                       "side": "interlocutor",
-                      "text": "I want to encourage you throughout your mission",
+                      "text": "One sec ...",
+                      "functionName": "registerUser",
                       "next": {
                         "type": "textMessage",
                         "side": "interlocutor",
-                        "text": "I need to let you know how great a job you are doing and notify you to keep going. Okay?",
+                        "text": "I, also, want to encourage you throughout your mission",
                         "next": {
-                          "type": "buttonGroupAnswer",
-                          "side": "user",
-                          "options": [
-                            {
-                              "type": "textMessage",
-                              "side": "user",
-                              "text": "Not right now",
-                              "answerText": "Not right now",
-                              "amplitudeEvent": "custom_event_demo_name",
-                              "next": {
+                          "type": "textMessage",
+                          "side": "interlocutor",
+                          "text": "I need to let you know how great a job you are doing and notify you to keep going. Okay?",
+                          "next": {
+                            "type": "buttonGroupAnswer",
+                            "side": "user",
+                            "options": [
+                              {
                                 "type": "textMessage",
-                                "side": "interlocutor",
-                                "text": "That's ok. I'll still try and help as much as I can. Let me show you what I found walking round the city today",
+                                "side": "user",
+                                "text": "Not right now",
+                                "answerText": "Not right now",
+                                "amplitudeEvent": "custom_event_demo_name",
```

Comment:
Is this truly the amplitude event name we want for this?

Reply:
we dont use this event, ill delete it, but for the record, this was already there, it shows up bc it moved down a couple rows since i inserted a new chat message